apps/web/src/pages/Auth/RegistrationPages/GettingStartedPage/GettingStartedForm.stories.tsx

```diff
@@ -25,6 +25,8 @@ const Template: StoryFn<typeof GettingStartedForm> = () => {
     <GettingStartedForm
       cacheKey=""
       query={mockFragmentData}
+      // NOTE: Needed for function colouring
```

Comment:
Could you please help me understand this change?

Reply:
Yeah, this prop requires an async function be passed to it. However, since `action` is not async, we just give it the keyword to colour it to satisfy typescript. Just an annoying bit of JS/TS. Since this is a story, I opted to ignore it instead of hacking in some async code just to satisfy typescript.